OpenRA.Mods.Common/Traits/Render/Contrail.cs

```diff
@@ -33,11 +33,29 @@ public class ContrailInfo : ConditionalTraitInfo, Requires<BodyOrientationInfo>
 		[Desc("Width of the trail.")]
 		public readonly WDist TrailWidth = new WDist(64);
 
-		[Desc("RGB color of the contrail.")]
-		public readonly Color Color = Color.White;
```

Comment:
This and the other renames will need an update rule.

Reply:
Should be fixed